components/connector/config/components.go

```diff
@@ -50,6 +54,11 @@ func InitInternalComponents(cfg Config, k8sClientset kubernetes.Interface) (Comp
 	)
 
 	certLoader := certificates.NewCertificateLoader(certCache, secretsRepository, caSecretName, rootCASecretName)
+	revocationListLoader := revocation.NewRevocationListLoader(revocationListCache,
+		k8sClientset.CoreV1().ConfigMaps(configmapNamespace.Namespace),
+		configmapNamespace.Name,
+		time.Second,
```

Comment:
Does it makes sense this to be configurable?

Reply:
It is needed because of the tests. It can be made configurable anyway.